build.gradle

```diff
@@ -16,45 +16,52 @@
  */
 
 plugins {
-    id 'net.wooga.plugins' version '1.4.0'
+    id 'net.wooga.plugins' version '3.0.0-rc.2'
 }
 
 group 'net.wooga.gradle'
 description = 'Node release plugin for Gradle.'
 
 repositories {
-    jcenter()
     mavenCentral()
     maven {
         url "https://plugins.gradle.org/m2/"
     }
 }
 
 dependencies {
-    testCompile('org.jfrog.artifactory.client:artifactory-java-client-services:2.+') {
+    testImplementation('org.jfrog.artifactory.client:artifactory-java-client-services:2.9.2') {
```

Comment:
Why locking this?

Reply:
No reason. I think I tried out IDEA's suggestion for version while upgrading the plugins.

Reply:
I will unlock it

Reply:
Still not unlocked :)

Reply:
I forgot to bring this up. I tried unlocking it but then the build failed with an error. I hadn't seen before. Could you try it on your end and see if it happens to you as well?

Reply:
```
Execution failed for task ':compileIntegrationTestGroovy'.
> org.apache.http.ssl.SSLContextBuilder

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

Reply:
You mean when using `2.+` or `(2.9.2,3]`?

Reply:
When using 2.+, 2+. 
I will try the range.